### PR TITLE
CHORE: Do not use pass in an except statement.

### PR DESCRIPTION
**Pull request by MaxJPRey**

### CODE REVIEW

File: pyaedt/modeler/cad/Primitives.py [hunk 1]
```diff
@@ -1231,13 +1231,13 @@ def _get_coordinates_data(self):  # pragma: no cover
                                                     coord.append(FaceCoordinateSystem(self, props, name))
                                                     break
                 except Exception:
```
Comment (MaxJPRey) [inline pyaedt/modeler/cad/Primitives.py:1233]:
```suggestion
                except Exception as e:
```

**Comment by SMoraisAnsys** [comment]
There seem to be an issue introduced by one of the last commits.
Since I'm also doing some refactoring, I adopted the follow approach about logging. If the method/function concerned is not public, I log in debug, else I use info. Since you mainly use error level, which approach do you think we should follow ?

**Comment by MaxJPRey** [comment]
> There seem to be an issue introduced by one of the last commits. Since I'm also doing some refactoring, I adopted the follow approach about logging. If the method/function concerned is not public, I log in debug, else I use info. Since you mainly use error level, which approach do you think we should follow ?

What issue did you noticed @SMoraisAnsys ?

**Comment by SMoraisAnsys** [comment]
> > There seem to be an issue introduced by one of the last commits. Since I'm also doing some refactoring, I adopted the follow approach about logging. If the method/function concerned is not public, I log in debug, else I use info. Since you mainly use error level, which approach do you think we should follow ?
> 
> What issue did you notice @SMoraisAnsys ?

I was speaking about the following:
image
Everything seems fine now

**Review by Samuelopez-ansys — DISMISSED** [review]

File: pyaedt/desktop.py [hunk 2]
```diff
@@ -945,7 +945,7 @@ def _initialize(
                             self.is_grpc_api = True
                             last_session.parent_desktop_id.append(self.aedt_process_id)
                             return True
-                    except:
+                    except Exception:
                         messages = desktop.GetMessages("", "", 0)
                         for message in messages:
                             if " GRPC server running on port: " in message and str(port) in message:
```
Comment (PipKat) [inline pyaedt/desktop.py:951]: gRPC is the standard way to capitalize thise term.

File: pyaedt/generic/touchstone_parser.py [hunk 3]
```diff
@@ -267,7 +267,7 @@ def get_return_loss_index(self, excitation_name_prefix=""):
         Returns
         -------
         list
-            list of index couples representing Return Losses of excitations
+            list of index couples representing Return Losses of excitations.
```
Comment (PipKat) [inline pyaedt/generic/touchstone_parser.py:270]:
```suggestion
            List of index couples representing return losses of excitations.
```

File: pyaedt/generic/touchstone_parser.py [hunk 4]
```diff
@@ -545,7 +545,7 @@ def check_touchstone_files(folder="", passivity=True, causality=True):
                     causality_check = float(msg_log.split("Maximum causality error: ")[-1].split("for entry")[0])
                     if not causality_check == 0.0:
                         is_causal = False
-                except:
+                except Exception:
                     is_causal = False
                     raise Exception("Failed evaluating causality value")
```
Comment (PipKat) [inline pyaedt/generic/touchstone_parser.py:550]:
```suggestion
                    raise Exception("Failed evaluating causality value.")
```
Messages should have verb (Failed) and also concluding punctuation.

File: pyaedt/modules/MeshIcepak.py [hunk 5]
```diff
@@ -341,7 +341,7 @@ def create(self, padding_values, padding_types, region_name, parts):
          Returns
         -------
         bool
-            True if successful, else False
+            ``True`` if successful, else ``False``.
```
Comment (PipKat) [inline pyaedt/modules/MeshIcepak.py:344]:
```suggestion
            ``True`` when successful, ``False`` when failed.
```
The suggested edit is the standard phrasing.

File: pyaedt/modules/MeshIcepak.py [hunk 6]
```diff
@@ -365,7 +365,7 @@ def delete(self):
...
```
Comment (PipKat) [inline pyaedt/modules/MeshIcepak.py:368]:
```suggestion
           ``True`` when successful, ``False`` when failed.
```

File: pyaedt/modules/MeshIcepak.py [hunk 7]
```diff
@@ -541,8 +541,8 @@ class MeshRegionCommon(object):
             Name of the mesh region.
         manual_settings : bool
             Whether to use manual settings or automatic ones.
```
Comment (PipKat) [inline pyaedt/modules/MeshIcepak.py:543]: How does one know how to set this parameter? I'd recommend making it clear by rewriting something like this: Whether to use manual settings. If ``False``, automatic settings are used.

File: pyaedt/desktop.py (hunk 2 above)
Comment (MaxJPRey) [inline pyaedt/desktop.py:951]:
```suggestion
                            if "gRPC server running on port: " in message and str(port) in message:
```

**Comment by MaxJPRey** [comment]
> > > There seem to be an issue introduced by one of the last commits. Since I'm also doing some refactoring, I adopted the follow approach about logging. If the method/function concerned is not public, I log in debug, else I use info. Since you mainly use error level, which approach do you think we should follow ?
> > 
> > 
> > What issue did you notice @SMoraisAnsys ?
> 
> I was speaking about the following: image Everything seems fine now

OK. Got it. Thanks @SMoraisAnsys

**Comment by MaxJPRey** [comment]
Thanks @PipKat for all the suggestions. All approved and/or committed.

File: pyaedt/modeler/cad/Primitives.py [hunk 8]
```diff
@@ -846,13 +846,13 @@ def _refresh_all_ids_from_aedt_file(self):
                 if operations and isinstance(operations.get("Operation", None), (OrderedDict, dict)):
                     try:
                         pid = operations["Operation"]["ParentPartID"]
-                    except Exception:  # pragma: no cover
-                        pass
+                    except Exception as e:  # pragma: no cover
+                        self.logger.error(e)
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/cad/Primitives.py:850]:
```suggestion
                        self.logger.debug(e)
```

File: pyaedt/modeler/cad/Primitives.py [hunk 9]
```diff
@@ -846,13 +846,13 @@ def _refresh_all_ids_from_aedt_file(self):
...
                 elif operations and isinstance(operations.get("Operation", None), list):
                     try:
                         pid = operations["Operation"][0]["ParentPartID"]
-                    except Exception:
-                        pass
+                    except Exception as e:
+                        self.logger.error(e)
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/cad/Primitives.py:855]:
```suggestion
                        self.logger.debug(e)
```

File: pyaedt/modeler/cad/Primitives.py [hunk 10]
```diff
@@ -1230,14 +1230,14 @@ def _get_coordinates_data(self):  # pragma: no cover
                                                     props = iop["FaceCSParameters"]
                                                     coord.append(FaceCoordinateSystem(self, props, name))
                                                     break
-                except Exception:
-                    pass
+                except Exception as e:
+                    self.logger.error(e)
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/cad/Primitives.py:1234]:
```suggestion
                    self.logger.debug(e)
```

File: pyaedt/modeler/cad/Primitives.py [hunk 11]
```diff
@@ -1230,14 +1230,14 @@ def _get_coordinates_data(self):  # pragma: no cover
...
             for cs in coord:
                 if isinstance(cs, CoordinateSystem):
                     try:
                         cs._ref_cs = id2name[name2refid[cs.name]]
-                    except Exception:
-                        pass
+                    except Exception as e:
+                        self.logger.error(e)
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/cad/Primitives.py:1240]:
```suggestion
                        self.logger.debug(e)
```

File: pyaedt/modeler/cad/Primitives.py [hunk 12]
```diff
@@ -8275,7 +8275,7 @@ def _create_object(self, name, pid=0, use_cached=False, is_polyline=False, **kwa
                     except Exception:
                         self.logger.warning("Unable to assign " + str(k) + " to object " + o.name + ".")
                 else:
-                    self.logger.error("'" + str(k) + "' is not a valid property of the primitive ")
+                    self.logger.error("'" + str(k) + "' is not a valid property of the primitive.")
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/cad/Primitives.py:8278]:
```suggestion
                    self.logger.debug("'" + str(k) + "' is not a valid property of the primitive.")
```

File: pyaedt/modules/MeshIcepak.py (hunk 7 above)
Comment (SMoraisAnsys) [inline pyaedt/modules/MeshIcepak.py:543]:
```suggestion
            Whether to use manual settings. If ``False``, automatic settings are used.
```

**Review by SMoraisAnsys — APPROVED** [review]
LGTM

**Pull request merged**